src/client/conn.rs

```diff
@@ -7,6 +7,41 @@
 //!
 //! If don't have need to manage connections yourself, consider using the
 //! higher-level [Client](super) API.
+//!
+//! ## Example
+//! A simple example that uses the `SendRequest` struct to talk HTTP over a Tokio TCP stream
+//! ```no_run
+//! # #[cfg(all(feature = "client", feature = "http1", feature = "runtime"))]
+//! # mod rt {
+//! use http::{Request, StatusCode};
+//! use hyper::{client::conn::Builder, Body};
+//! use tokio::net::TcpStream;
+//!
+//! #[tokio::main]
+//! async fn main() {
+//!     let target_stream = TcpStream::connect("example.com:80").await.unwrap();
+//!
+//!     let handshake = Builder::new()
+//!         .handshake::<TcpStream, Body>(target_stream)
+//!         .await.unwrap();
+//!
+//!     // spawn a task to poll the connection and drive the HTTP state
+//!     let connection = handshake.1;
+//!     tokio::spawn(connection.without_shutdown());
```

Comment:
I don't think the `without_shutdown` is required for the example. Perhaps if just spawning into the background, we could show printing an error if it occurs, whatcha think?